pkg/kubelet/kubelet.go

```diff
@@ -1946,7 +1946,7 @@ func (kl *Kubelet) SyncPod(ctx context.Context, updateType kubetypes.SyncPodType
 
 	// If the pod is terminal, we don't need to continue to setup the pod
 	if apiPodStatus.Phase == v1.PodSucceeded || apiPodStatus.Phase == v1.PodFailed {
-		kl.statusManager.SetPodStatus(pod, apiPodStatus)
+		kl.statusManager.SetPodStatus(klog.TODO(), pod, apiPodStatus)
```

Comment:
```suggestion
		kl.statusManager.SetPodStatus(klog.FromContext(ctx), pod, apiPodStatus)
```

Reply:
or even better - do `logger := klog.FromContext(ctx)` somewhere above and use it here.